code/src/components/Header.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import styled from 'styled-components'
+import headerImage from 'assets/headerImage.png'
+
+const date = new Date();
+const todaysDate = date.toString().slice(0, 15);
+
+const Header = () => {
+  return (
+    <HeaderContainer>
+      <TodaysDate>{todaysDate}</TodaysDate>
+      <HeaderText>Todays todo list!</HeaderText>
+      <HeaderImage
+        src={headerImage}
```

Comment:
Love the theme you went with. Great use of every color from the picture for different components and states! :D